test/test_helpers.js

```diff
@@ -112,8 +112,9 @@ function getMockEnvironmentAsync(options) {
     });
 
     // istanbul ignore next
-    // eslint-disable-next-line no-unused-vars
```

Comment:
`// eslint-disable-next-line no-unused-vars` was not working on my system. `// eslint-disable-next-line` without any rules worked. So I switch to `/* eslint-disable no-unused-vars */` here to be able to disable the specific rule.